internal/internal_nexus_task_handler.go

```diff
@@ -469,16 +500,13 @@ var emptyReaderNopCloser = io.NopCloser(bytes.NewReader([]byte{}))
 
 // convertKnownErrors converts known errors to corresponding Nexus HandlerError.
 func convertKnownErrors(err error) error {
-	// Handle common errors returned from various client methods.
-	if workflowErr, ok := err.(*WorkflowExecutionError); ok {
-		return nexus.NewFailedOperationError(workflowErr)
-	}
-	if queryRejectedErr, ok := err.(*QueryRejectedError); ok {
-		return nexus.NewFailedOperationError(queryRejectedErr)
-	}
 	// Not using errors.As to be consistent ApplicationError checking with the rest of the SDK.
 	if appErr, ok := err.(*ApplicationError); ok && appErr.NonRetryable() {
-		return nexus.NewFailedOperationError(appErr)
+		return &nexus.HandlerError{
+			// TODO(bergundy): Change this to a non retryable internal error after the 1.27.0 server release.
```

Comment:
Can we create to track this? Also doesn't this need to be done before GA since it is a breaking change?

Reply:
https://github.com/temporalio/sdk-go/issues/1810